lib/modules/betteReddit.js

```diff
@@ -219,8 +219,12 @@ module.go = () => {
 function commentsLinksNewTabs(ele = document.body) {
 	const links = ele.querySelectorAll('.thing div.md a');
 	links::forEachChunked(link => {
-		link.target = '_blank';
-		link.rel = 'noopener noreferer';
+		const { baseURI, href } = link;
+
+		if (!(href.includes(baseURI) && (href.includes('#!settings') || href.includes('#res:settings')))) {
```

Comment:
Sounds good! Will make the first two changes.

With ```if (href.includes(baseURI) && isSettingsUrl(href))``` I check for changes in location and only then checks for the settings hash. 
So in that case, responses from the AutoModerator with say a link to settings at ```https://www.reddit.com/#!settings/search``` still open in a new tab.
 

Is this what the comment described? Can you give me a more concrete example on Reddit so I can test this?